src/Symfony/Component/Translation/Translator.php

```diff
@@ -86,7 +88,7 @@ class Translator implements LegacyTranslatorInterface, TranslatorInterface, Tran
     /**
      * @throws InvalidArgumentException If a locale contains invalid characters
      */
-    public function __construct(?string $locale, MessageFormatterInterface $formatter = null, string $cacheDir = null, bool $debug = false)
+    public function __construct(?string $locale, MessageFormatterInterface $formatter = null, string $cacheDir = null, bool $debug = false, array $cacheVary = [])
```

Comment:
I was looking at this again an I'm not sure if we can add a new constructor arg on 4.3?

Reply:
As long as it's added with a default value at the end, I think yes.

Reply:
Although it would look like a new feature.